### PR TITLE
custom repo url 

### PR DESCRIPTION
**Pull request by tawfekov**
add your custom own packgist url you want to mirror , default value from DI

### CODE REVIEW

File: src/Command/DumpMetadataCommand.php [hunk 1]
```diff
@@ -26,20 +27,24 @@ final class DumpMetadataCommand extends Command
...
 
     protected function configure()
     {
+        $this->addArgument('url', InputArgument::OPTIONAL, 'what is the url of the repo you want to mirror ?');
         $this->setDescription('Dump metadata files.');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $url = $input->getArgument('url') ? $input->getArgument('url') : $this->url;
```
Comment (IndraGunawan) [inline src/Command/DumpMetadataCommand.php]: could you please use `addArgument` default value. https://github.com/symfony/symfony/blob/4.0/src/Symfony/Component/Console/Command/Command.php#L367

**Review by IndraGunawan — CHANGES_REQUESTED** [review]
Thanks for your PR.
have you test your code?

File: src/Command/DumpMetadataCommand.php (hunk 1 above)
Comment (tawfekov) [reply src/Command/DumpMetadataCommand.php]: @IndraGunawan I've resolved this , can you please merge it ?

**Comment by tawfekov** [comment]
Hi @IndraGunawan , I've updated it 5 days ago , can you please merge it ?

**Comment by IndraGunawan** [comment]
thanks @tawfekov
sorry for late response

**Pull request merged**